### PR TITLE
 add minimum files to get started

### PR DESCRIPTION
**Pull request by deads2k**
/assign @sttts

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+language: go
+
+go:
+  - 1.9
```
Comment (sttts) [inline .travis.yml]: 1.10

File: OWNERS [hunk 2]
```diff
@@ -0,0 +1,11 @@
+reviewers:
```
Comment (sttts) [inline OWNERS]: file seems off

File: cluster-kube-apiserver-operator.spec [hunk 3]
```diff
@@ -0,0 +1,142 @@
+#
```
Comment (sttts) [inline cluster-kube-apiserver-operator.spec]: do we really need rpms?

**Comment by sttts** [comment]
Some comments. Follow-ups are fine.

How do we get prow?

File: cluster-kube-apiserver-operator.spec (hunk 3 above)
Comment (deads2k) [reply cluster-kube-apiserver-operator.spec]: > do we really need rpms?

don't rock the boat yet.

**Comment by deads2k** [comment]
comments addressed

**Comment by deads2k** [comment]
> Some comments. Follow-ups are fine.
> 
> How do we get prow?

If you're good with it, use the green button.

Prow is a good question.

**Pull request merged**